ert_shared/models/base_run_model.py

```diff
@@ -190,55 +190,6 @@ def get_runtime(self) -> Union[int, float]:
     def is_forward_model_finished(progress) -> bool:
         return all(job.status == "Success" for job in progress)
 
-    def update_progress_for_index(self, iteration: int, idx, run_arg: RunArg) -> None:
-        try:
-            # will throw if not yet submitted (is in a limbo state)
-            queue_index = run_arg.getQueueIndex()
-        except (ValueError, AttributeError):
-            return
-
-        status = None
-        timed_out = False
-        if self._job_queue:
-            status = self._job_queue.getJobStatus(queue_index)
-            timed_out = self._job_queue.did_job_time_out(queue_index)
-
-        # Avoids reading from disk for jobs in these states since there's no
-        # data anyway. If timed out, never exit here as that would prevent
-        # propagation of the failure status.
-        if (
-            status
-            in [
-                JobStatusType.JOB_QUEUE_PENDING,
-                JobStatusType.JOB_QUEUE_SUBMITTED,
-                JobStatusType.JOB_QUEUE_WAITING,
-            ]
-            and not timed_out
-        ):
-            return
-
-        fms = self.realization_progress[iteration].get(run_arg.iens, None)
-        jobs = fms[0] if fms else None
-
-        # Don't load from file if you are finished
-        if not fms or not BaseRunModel.is_forward_model_finished(fms[0]):
```

Comment:
Might also make `is_forward_model_finished` superfluous.

Reply:
It's used here
https://github.com/equinor/ert/blob/7d0254f5de6b6d4e1df4774c51962f07ab62474e/tests/ert_tests/models/test_base_run_model.py#L32-L33
With my (limited) insight into the details of the queue-mechanism, this seems like a reasonable test. No...?

Reply:
Yes, you are right!